engine/installation/linux/docker-ee/rhel.md

```diff
@@ -27,12 +27,14 @@ Docker Community Edition (Docker CE) is not supported on {{ linux-dist-long }}.
 
 ### OS requirements
 
-To install Docker EE, you need the 64-bit version of {{ linux-dist-long }} 7
-running on an x86 hardware platform, or `s390x` (IBM Z) architecture.
-
-In addition, you must use the `devicemapper` storage driver. On production
-systems, you must use `direct-lvm` mode, which requires one or more dedicated
-block devices. Fast storage such as solid-state media (SSD) is recommended.
+To install Docker EE, you need the 64-bit version of {{ linux-dist-long }}
+running on `x86_64`, `s390x` (IBM Z), or `ppc64le` (IBM POWER) architectures.
+
+In addition, you must use the `overlay2` or `devicemapper` storage driver.
+**The `overlay2` storage driver is only supported on RHEL 7.4 or higher.** On
```

Comment:
@dmp42 PTAL

Reply:
@thaJeztah tells me that `overlay2` will correctly work on RHEL 7.2+ now. Is this true?

Reply:
It will (at least on the upcoming release - don't know if it was decided on to backport to 17.06-ee